src/libaktualizr/package_manager/androidmanager.cc

```diff
@@ -0,0 +1,127 @@
+#include <boost/algorithm/string/find.hpp>
+#include <boost/filesystem.hpp>
+#include <forward_list>
+
+#include "androidmanager.h"
+
+#include "utilities/utils.h"
+
+#include <boost/phoenix/stl/algorithm/transformation.hpp>
+#include <boost/spirit/include/phoenix_container.hpp>
+#include <boost/spirit/include/phoenix_core.hpp>
+#include <boost/spirit/include/qi.hpp>
```

Comment:
What was it that required using these boost libraries? I see the code below, but I'm curious while normal C++ string parsing was inadequate.

Reply:
I'm not aware of all the approaches you're trying to follow in the project, so be specific. Standard library does not offer comparable functionality, and hand-written find/extract would be less expressive than using boost/spirit/Qi.

Reply:
No problem, I think I've got a good enough handle on it.